mmdet3d/models/decode_heads/cylinder3d_head.py

```diff
@@ -86,72 +45,91 @@ def build_conv_seg(self, channels: int, num_classes: int,
             padding=1,
             bias=True)
 
-    def forward(self, sparse_voxels: SparseConvTensor) -> SparseConvTensor:
+    def forward(self, voxel_dict: dict) -> dict:
         """Forward function."""
-        sparse_logits = self.cls_seg(sparse_voxels)
-        return sparse_logits
+        sparse_logits = self.cls_seg(voxel_dict['voxel_feats'])
+        voxel_dict['logits'] = sparse_logits.features
```

Comment:
Should we put `logits` into `voxel_dict`?

Reply:
We will use 'inverse_map' and some other information in `voxel_dict` to map `voxel_logits` to `point_logits` in `predict` procedure. It is better to return `voxel_dict` here rather than `logits`.

Reply:
If we want to put this into `voxel_dict`, maybe a comment is needed